app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import datetime
+
+from app.errors import NotVaccinatedError, NotWearingMaskError
+
+
+def go_to_cafe(friends, cafe):
```

Comment:
annotation is love

Reply:
it won't me allow me to add the annotation for cafe, not sure why. https://prnt.sc/SVI0RiUTqRns